services/autoscaling/tests/manual/.env-devel

```diff
@@ -3,8 +3,9 @@ AUTOSCALING_LOGLEVEL=INFO
 AUTOSCALING_TASK_INTERVAL=30
 AUTOSCALING_EC2_ACCESS_KEY_ID=XXXXXXXXXX
 AUTOSCALING_EC2_SECRET_ACCESS_KEY=XXXXXXXXXX
-AUTOSCALING_EC2_ENDPOINT=https://ec2.amazonaws.com
+AUTOSCALING_EC2_ENDPOINT=null
```

Comment:
Could you please update these separately on  `deployments/osparc.local/repo.config` in the osparc-config repo? This way we keep them reasonably in sync

Reply:
euh? this is not the main .env-devel man. but the .env-devel for manual testing used when you run directly in AWS.

Reply:
and they are already updated as far as I know. and they are only useful in an AWS deploy.